src/app/page.js

```diff
@@ -32,27 +34,36 @@ export default function HelpsPage() {
       />
       <Container maxWidth="xl">
         <Stack spacing={2}>
-          <div
-            style={{
-              display: "flex",
-              flexDirection: "row",
-              justifyContent: "space-between",
-              alignItems: "center",
-              marginTop: 50,
-            }}
+          <Grid
+            container
+            direction={isXsScreen ? "column-reverse" : "row"}
+            justifyContent="space-between"
+            alignItems="flex-start"
+            style={{ marginTop: 50 }}
           >
             <Typography variant="h3">قائمة الطلبات</Typography>
-            <Link href={"/helps/create"}>
-              <Button color="primary" variant="contained" size="large">
-                <Typography variant="h6" color={"white"}>
-                  إضافة طلب
-                </Typography>
-              </Button>
-            </Link>
-          </div>
+            <Grid item style={{ width: isXsScreen ? "100%" : "auto" }}>
+              <Link href={"/helps/create"}>
+                <div style={{ width: isXsScreen ? "100%" : "auto" }}>
+                  <Button
+                    color="error"
+                    variant="contained"
+                    size="large"
+                    startIcon={<AddIcon />}
+                    style={{ width: "100%", marginBottom: isXsScreen ? "20px" : "0px" }}
+                  >
+                    <Typography variant="h6" color={"white"}>
+                      إضافة طلب
+                    </Typography>
+                  </Button>
+                </div>
+              </Link>
+            </Grid>
+          </Grid>
           <HelpsFilter setFilters={setHelpsFilter} />
         </Stack>
-        <Grid container>
+        {/* These Grid components might be optional, feel free to check if we still need them! */}
```

Comment:
I tested it and it has no impact, it would be nice if you clean it.
Thank you 🙏

Reply:
Done! 🚀